package.json

```diff
@@ -9,14 +9,15 @@
     "atom"
   ],
   "scripts": {
-    "eslint": "eslint ./",
+    "eslint": "eslint ./lib",
```

Comment:
Why do you add again `./lib` ?
If you need to exclude a directory, put in in the `.eslintignore` file.